langkit/templates/pkg_implementation_body_ada.mako

```diff
@@ -3986,6 +3986,20 @@ package body ${ada_lib_name}.Implementation is
       end;
    end Reroot_Foreign_Node;
 
+   function Text
+     (Node : access ${root_node_value_type}'Class) return ${T.String.name}
+   is
+      T : Text_Type := Text (Node);
+      Result : constant ${T.String.name} :=
+         ${T.String.constructor_name} (T'Length);
+   begin
+      for I in T'Range loop
+         Result.Items (I - T'First + Result.Items'First) :=
+            ${T.String.element_type.to_internal_expr('T (I)', 'Context')};
```

Comment:
Same comment as in previous commit: could’n the loop be replaced with direct assignment?